tests/test_host.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from illuminatio.host import (
+    ClusterHost,
+    ConcreteClusterHost,
+    ExternalHost,
+    LocalHost,
+    GenericClusterHost,
+    Host,
+)
+
+
+@pytest.mark.parametrize(
+    "identifier,expected_host",
+    [
+        pytest.param("localhost", LocalHost(), id="LocalHost"),
+        pytest.param(
+            "127.0.0.1", ExternalHost("127.0.0.1"), id="ExternalHost with localhost IP"
+        ),
+        pytest.param(
+            "123.123.123.123",
```

Comment:
Could you also add a test with an IPv6 address, just to ensure we don't break anything when we introduce IPv6